### PR TITLE
chore: Update external link account parameters

### PR DESCRIPTION
**Pull request by alexzchut**
Update external link account parameters

[Monday ticket](add your monday ticket url here)

`Please describe PR here`

<details>
  <summary>Platforms Affected</summary>
  
- [ ] Android
- [ ] Android TV
- [ ] Apple TV (TVOS)
- [ ] Fire TV (FireOS)
- [ ] iOS
- [ ] LG TV (WebOS)
- [ ] Samsung TV (Tizen)
- [ ] Web
</details>

## Known or Pending Issues

`fill in this section with potential issues / limitations the reviewer(s) should know about`

## Screenshots

`If there is some kind of ui change please share screenshots`

## Version Info

`Does this update require a certain version of quickbrick, or does it rely on another pr to work?`

### CODE REVIEW

**Comment by alexzchut** [comment]
@ranmeirman this is the PR where I removed previously added parameters to the storefront plugin and adding them to the login flow, @kutsko-applicaster will add more changes on the login flow plugin to pull the values from itself instead of going to other plugin

File: plugins/quick-brick-storefront/manifests/manifest.config.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ const iAndroid = R.includes(R.__, androidPlatforms);
 const isWeb = R.includes(R.__, webPlatforms);
 
 const min_zapp_sdk = {
-  ios_for_quickbrick: "6.7.0-Dev",
+  ios_for_quickbrick: "5.8.0-Dev",
```
Comment (ranmeirman) [inline plugins/quick-brick-storefront/manifests/manifest.config.js:32]: why this?

Comment (alexzchut) [reply plugins/quick-brick-storefront/manifests/manifest.config.js:32]: @ranmeirman this was a revert of the changes made in this plugin so also this was reverted..

Comment (ranmeirman) [reply plugins/quick-brick-storefront/manifests/manifest.config.js:32]: but the plugin was published I assume

Comment (alexzchut) [reply plugins/quick-brick-storefront/manifests/manifest.config.js:32]: @ranmeirman no, I removed the published version

**Pull request merged**